conf/app.routes

```diff
@@ -15,3 +15,5 @@ POST       /subsidy/update                                  uk.gov.hmrc.eusubsid
 POST       /undertaking/disable                             uk.gov.hmrc.eusubsidycompliance.controllers.UndertakingController.disableUndertaking()
 
 POST       /subsidy/retrieve                                uk.gov.hmrc.eusubsidycompliance.controllers.UndertakingController.retrieveSubsidies()
+
+POST       /send-nudge-email                                uk.gov.hmrc.eusubsidycompliance.controllers.EmailController.sendNudgeEmail()
```

Comment:
I think the expected url is /email-notification. Please confirm, I think its mentioned in the specs

Reply:
Have checked this, correct it is /email-notification good spot!